dvc/command/tag.py

```diff
@@ -1,6 +1,8 @@
-import yaml
 import logging
 
+from ruamel.yaml import YAML
+from ruamel.yaml.compat import StringIO
```

Comment:
we have our own implementation in `dvc.utils.compat`. I think, it's better to avoid relying on some internal stuff from a third party library.